tests/flow-pkt-recursion/flow-pkt-recursion-terminated-excluded/test.yaml

```diff
@@ -0,0 +1,19 @@
+# Exclude recursion-level from flow matching
+# Re-use the pcap from the middleware-included test.
+command: |
+  ${SRCDIR}/src/suricata -c "${SRCDIR}/suricata.yaml" -l "${OUTPUT_DIR}" \
+   -r "${TEST_DIR}/../flow-pkt-recursion-terminated-included/test.pcap" \
+   --set "threshold-file=${SRCDIR}/threshold.config" \
+   --set "classification-file=${SRCDIR}`[ -f ${SRCDIR}/etc/classification.config ] && printf '/etc'`/classification.config" \
+   --set "reference-config-file=${SRCDIR}`[ -f ${SRCDIR}/etc/reference.config ] && printf '/etc'`/reference.config" \
+   --set 'decoder.recursion-level.use-for-tracking=false'
```

Comment:
You can use `args` instead of the whole command here

(and `pcap:` to reuse another pcap)